PROTOCOL.md

```diff
@@ -253,7 +253,7 @@ The following is an example `remove` action.
 
 ### Transaction Identifiers
 Incremental processing systems (e.g., streaming systems) that track progress using their own application-specific versions need to record what progress has been made, in order to avoid duplicating data in the face of failures and retries during a write.
-Transaction identifiers allow this information to be recorded atomically in the transaction log of a delta table along with the other actions that modify the contents of the table.
+Transaction identifiers allow this information to be recorded atomically in the transaction log of a Delta table along with the other actions that modify the contents of the table.
```

Comment:
This change is valid, but the other changes in this file refer to delta (like change) files (which contain incremental information about commits to the Delta table), not Delta table files. I think those should stay lowercase.

Reply:
Fixed.